src/psyclone/tests/test_files/dynamo0p3/testkern_multi_anyw2.f90

```diff
@@ -1,57 +0,0 @@
-! Copyright (c) 2017-2019, Science and Technology Facilities Council
```

Comment:
It's nicer if you (remember to) use `git mv` when renaming files.

Reply:
I did! I always use `git mv` when renaming files and it is not clear to me why it's so poorly represented in the diffs - very frustrating.

Reply:
Annoying!